plugin/trino-clickhouse/src/test/java/io/trino/plugin/clickhouse/TestClickHouseConnectorTest.java

```diff
@@ -845,6 +846,32 @@ public void testTextualPredicatePushdown()
                 .isFullyPushedDown();
     }
 
+    @Test
+    public void testPredicateOverTopN()
```

Comment:
I expect these tests are already covered by BaseJdbcConnectorTest.testTopNPushdown. Please remove it if so. 

Note that we should update the base test unless the test is specific to ClikcHouse connector.

Reply:
Removed the tests. The ones in `testTopNPushdown` check the same, thanks.